File: src/main/java/ss/fortberg/util/JsonUtils.java
package ss.fortberg.util;

import com.fasterxml.jackson.databind.DeserializationFeature;
import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.SerializationFeature;

public class JsonUtils {

    public static final ObjectMapper objectMapper = new ObjectMapper()
        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
        .enable(SerializationFeature.INDENT_OUTPUT);
}

File: src/main/kotlin/ss/fortberg/util/JsonUtils.kt
package ss.fortberg.util

import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.databind.SerializationFeature
import com.fasterxml.jackson.module.kotlin.KotlinModule

val objectMapper = ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT)
    .registerModule(KotlinModule.Builder().build())